.gitattributes

```diff
@@ -0,0 +1,11 @@
+# ignored file when package to source.tgz
+.gitattributes export-ignore
+.gitignore export-ignore
```

Comment:
```suggestion
```

Reply:
see https://github.com/apache/incubator-hugegraph/blob/master/.gitattributes#L3

Reply:
> apache/incubator-hugegraph@`master`/.gitattributes#L3

forget the file doesn't have LICENSE header (if we add for it, we could keep it in source code)